rest-api/tools/auth_setup.sh

```diff
@@ -61,7 +61,7 @@ source ${REPO_ROOT_DIR}/tools/setup_vars.sh
 TMP_DB_INFO_FILE=${TMP_DIR}/db_info.json
 PORT=3308
 CLOUD_PROXY_PID=
-
+export "GOOGLE_APPLICATION_CREDENTIALS"=$CREDS_FILE
```

Comment:
Nit: I don't think you need the quotes here.

Reply:
You're right